cmd/tetragon/main.go

```diff
@@ -140,6 +140,14 @@ func tetragonExecute() error {
 	log.WithField("version", version.Version).Info("Starting tetragon")
 	log.WithField("config", viper.AllSettings()).Info("config settings")
 
+	if option.Config.ForceLargeProgs {
+		log.Info("Force loading large programs")
+	}
+
+	if option.Config.ForceSmallProgs {
+		log.Info("Force loading smallprograms")
+	}
+
```

Comment:
Should we maybe check that not both `ForceLargeProgs` and `ForceSmallProgs` is set and bail out if so?

Reply:
right, good idea, added, thanks